GTG/gtk/preferences.py

```diff
@@ -17,304 +17,50 @@
-            self.binding_backup = self.new_task_binding
-        else:
-            # There exists a shortcut
-            self.button.set_active(True)
-        (accel_key, accel_mods) = Gtk.accelerator_parse(self.new_task_binding)
-        self.show_input = Gtk.accelerator_get_label(accel_key, accel_mods)
-        self.liststore.set_value(iter1, 1, self.show_input)
-
-    def on_shortcut_toggled(self, widget):
-        """ New task shortcut checkbox is toggled """
-        if widget.get_active():
-            self.new_task_binding = self.binding_backup
-            shortcut.save_new_binding(self.new_task_binding, True)
-            self.cell.set_property("editable", True)
-        else:
-            self.new_task_binding = ""
-            shortcut.save_new_binding(self.new_task_binding, True)
-            self.cell.set_property("editable", False)
-
-    def _cellAccelEdit(self, cell, path, accel_key, accel_mods, code, model):
-        """ Accelerator is modified """
-        self.show_input = Gtk.accelerator_get_label(accel_key, accel_mods)
-        self.new_task_binding = Gtk.accelerator_name(accel_key, accel_mods)
-        if shortcut.check_invalidity(self.new_task_binding, accel_key,
-                                     accel_mods):
-            self._show_warning(self.show_input)
-            return
-        self.binding_backup = self.new_task_binding
-        iter = model.get_iter(path)
-        model.set_value(iter, 1, self.show_input)
-        shortcut.save_new_binding(self.new_task_binding,
-                                  self.button.get_active())
-
-    def _accel_cleared(self, widget, path, model):
-        """ Clear the accelerator """
-        iter = model.get_iter(path)
-        model.set_value(iter, 1, None)
-
-    def _show_gsettings_install_label(self):
-        vbox = self.builder.get_object("prefs-vbox7")
-        label = Gtk.Label()
-        label.set_markup(_("<small>Please install <i><b>gsettings</b></i> "
-                           "to enable New task shortcut</small>"))
-        vbox.add(label)
-
-    def _show_warning(self, input_str):
-        """ Show warning when user enters inappropriate accelerator """
-        show = _("The shortcut \"%s\" cannot be used because "
-                 "it will become impossible to type using this key.\n"
-                 "Please try with a key such as "
-                 "Control, Alt or Shift at the same time.") % input_str
-        dialog = Gtk.MessageDialog(self.dialog,
-                                   Gtk.DialogFlags.DESTROY_WITH_PARENT,
-                                   Gtk.MessageType.WARNING,
-                                   Gtk.ButtonsType.OK,
-                                   show)
-        dialog.run()
-        dialog.hide()
+    def on_sidebar_change(self, widget, data=None):
+        if self.stack.get_visible_child_name() == "general":
+            self.headerbar.set_title("General Preferences")
```

Comment:
hmm, I have tried the best solution that came to my mind:

self.headerbar.set_title(self.stack.get_visible_child().get_property('title'))

However, this "get_property('title')" tries to find the property not in the child declaration I'm doing here above, when adding the child to stack, but it is searching for the title in the GtkScrolledWindow widget itself which, sadly, cannot have 'title' as a property :/